testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksChangeDataFeedCompatibility.java

```diff
@@ -416,4 +435,74 @@ public void testDeleteFromNullPartitionWithCdfEnabled()
             onDelta().executeQuery("DROP TABLE IF EXISTS default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testThatEnableCdfTablePropertyIsShownForCtasTables()
+    {
+        String tableName = "test_show_create_show_property_for_table_created_with_ctas_" + randomNameSuffix();
+        try {
+            onTrino().executeQuery("CREATE TABLE delta.default." + tableName + "(page_url, views)" +
+                    "WITH (location = 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "', " +
+                    "change_data_feed_enabled = true) AS VALUES ('url1', 1), ('url2', 2)");
+            Assertions.assertThat(onTrino().executeQuery("SHOW CREATE TABLE " + tableName).getOnlyValue().toString())
+                    .contains("change_data_feed_enabled = true");
+        }
+        finally {
+            onTrino().executeQuery("DROP TABLE IF EXISTS delta.default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testThatCdfDoesntWorkWhenPropertyIsNotSet()
+    {
+        String tableName1 = "test_cdf_doesnt_work_when_property_is_not_set_1_" + randomNameSuffix();
+        String tableName2 = "test_cdf_doesnt_work_when_property_is_not_set_2_" + randomNameSuffix();
+        assertThereIsNoCdfFileGenerated(tableName1, "");
+        assertThereIsNoCdfFileGenerated(tableName2, "change_data_feed_enabled = false");
+    }
+
+    private void assertThereIsNoCdfFileGenerated(String tableName, String tableProperty)
+    {
+        try {
+            onTrino().executeQuery("CREATE TABLE delta.default." + tableName + " (col1 VARCHAR, updated_column INT) " +
+                    "WITH (location = 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                    (tableProperty.isEmpty() ? "" : ", " + tableProperty) + ")");
+
+            if (tableProperty.isEmpty()) {
+                Assertions.assertThat(onTrino().executeQuery("SHOW CREATE TABLE " + tableName).getOnlyValue().toString())
+                        .doesNotContain("change_data_feed_enabled = true");
```

Comment:
I think we should verify
```suggestion
                        .contains("change_data_feed_enabled = false");
```

Reply:
in that case we just don't want `change_data_feed_enabled` to be there at all. I will change to 
`doesNotContain("change_data_feed_enabled");`